organization/src/main/kotlin/com/cosmotech/organization/azure/OrganizationServiceImpl.kt

```diff
@@ -273,6 +271,22 @@ internal class OrganizationServiceImpl(
     return OrganizationAccessControl(rbacAccessControl.id, rbacAccessControl.role)
   }
 
+  override fun updateOrganizationAccessControl(
+      organizationId: String,
+      identityId: String,
+      organizationRole: OrganizationRole
+  ): OrganizationAccessControl {
+    val organization = findOrganizationById(organizationId)
+    csmRbac.verify(organization.getRbac(), PERMISSION_WRITE_SECURITY)
+    csmRbac.getAccessControl(organization.getRbac(), identityId)
+    val rbacSecurity =
+        csmRbac.setUserRole(organization.getRbac(), identityId, organizationRole.role)
+    organization.setRbac(rbacSecurity)
+    cosmosTemplate.upsertAndReturnEntity(coreOrganizationContainer, organization)
```

Comment:
You can stored and used the returned value from  `upsertAndReturnEntity` here.

something like this :
```
    val updatedOrganization = cosmosTemplate.upsertAndReturnEntity(coreOrganizationContainer, organization)
    val rbacAccessControl = csmRbac.getAccessControl(updatedOrganization.getRbac(), identityId)
    return OrganizationAccessControl(rbacAccessControl.id, rbacAccessControl.role)
```